DataFormats/PatCandidates/interface/MET.h

```diff
@@ -200,11 +247,25 @@ namespace pat {
       // ---- holder for pfMET specific info ---
       std::vector<SpecificPFMETData> pfMET_;
 
+      // MET significance
+      double metSig_;
+      
+      const PackedMETUncertainty findMETTotalShift(MET::METCorrectionLevel cor, MET::METUncertainty shift) const;
+   
+      std::map<MET::METCorrectionLevel, std::vector<MET::METCorrectionType> > corMap_;
+      void initCorMap();
+
+    
+
     protected:
 
       // ---- non-public correction utilities ----
+      //kept for 74X backward-compatibility, not initialized and used in 75X
```

Comment:
is this really about 74X vs 75X (code used to fill the values) or about different (class) versions of the pat::MET?
If it is for the class version, a better approach would be to write a root iorule to convert from the old MET format stored in "old" files to the new format.